src/ansys/dpf/composites/composite_model.py

```diff
@@ -125,15 +129,24 @@ class CompositeModel:
         the :class:`.ContinuousFiberCompositesFiles` object.
```

Comment:
Documentation for the `server` and `default_unit_system` parameters seems to be missing.